src/components/FindPark.vue

```diff
@@ -22,10 +23,16 @@
       type="text"
       aria-label="Type your search terms here"
       placeholder="Enter City, State to search"
+      aria-describedby="Enter city or address to search for specific parks near you."
       v-model="searchTerm"
     />
     <router-link v-if="searchTerm" to="/results">
-      <button id="search" class="search-button" @click="search">
+      <button 
+        id="search" 
+        class="search-button" 
+        @click="search"
+        aria-describedby="Click this button to search by city or address after filling out the input above."
+      >
```

Comment:
Oh, I've not used `aria-describedby` before. I'll have to check it out. Glad we have these labels for screen readers.

Reply:
Same here! I'm excited to look into this more in the future.  Thanks for considering this, Nathan.

Reply:
My pleasure! Tried to make it usable and accessible for all users with as many disabilities as possible.